modules/rbac-viewing-cluster-roles.adoc

```diff
@@ -23,11 +23,19 @@ endif::[]
 
 .Procedure
 
+:openshift-enterprise:
```

Comment:
Is this supposed to be part of an `ifdef`?

Reply:
Yikes, that was an addition to test in Atom preview, not sure how I missed removing it. Thanks for catching!